helpers/ifrauBackdropService.js

```diff
@@ -1,10 +1,13 @@
+import { isFramed } from './framed.js';
+
 let ifrauBackdropService;
 
 export async function tryGetIfrauBackdropService() {
-	if (!window.ifrauclient) return null;
+	if (!await isFramed()) return null;
 	if (ifrauBackdropService) return ifrauBackdropService;
 
-	const ifrauClient = await window.ifrauclient().connect();
+	const ifrau = await import('ifrau/client/slim.js');
+	const ifrauClient = await new ifrau.SlimClient().connect();
 	ifrauBackdropService = await ifrauClient.getService('dialogWC', '0.1');
```

Comment:
It seems a little strange that this backdrop specific helper is defined outside of dialog, even though the iframe service itself is dialog specific.  As written, being dialog specific, I would expect this to live with the dialog component.  If it were changed to be a more generic backdrop service, perhaps it could live with the backdrop component.  🤷‍♂️

Reply:
Yeah, I recalled it living in the dialog component at some point and was surprised to find that it wasn't when I went in to make this change. I didn't look into when/why it moved.

Reply:
Cool, killing two birds with one stone here then, I guess. :)